lib/nm/index.ts

```diff
@@ -288,8 +288,11 @@ export class NetworkManager extends NetworkManagerTypes {
 			if (_.isUndefined(wifiConnection)) {
 				return;
 			}
-			const {Id} = await this.getObjectProperty(['org.freedesktop.NetworkManager.Connection.Active', 'Id'], wifiConnection.path);
-			return Id;
+			const {Connection} = await this.getObjectProperty(['org.freedesktop.NetworkManager.Connection.Active', 'Connection'], wifiConnection.path);
```

Comment:
Line length is a bit excessive here, can we multi-line this? i.e.

```ts
const {Connection} = await this.getObjectProperty([
  'org.freedesktop.NetworkManager.Connection.Active',
  'Connection'
], wifiConnection.path);
```

Reply:
Yeah, not a fan too. Changed in the next commit

Reply:
It's actually something recurrent with all calls to `this.getObjectProperty`, so for now I'll split it but I'd like to refactor that in the future